blockchyp/blockchyp_crypto.py

```diff
@@ -0,0 +1,51 @@
+"""
+This file defines the BlockChyp Cryptography Services
```

Comment:
A module docstring probably shouldn't start with "this file". It should probably look something like:

```
"""Helpers for cryptographic communication in the BlockChyp client"""
```